src/Infrastructure.VS/TextDocumentProvider.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace SonarLint.VisualStudio.Infrastructure.VS
 {
-    public interface ITextDocumentProvider
+    internal interface ITextDocumentProvider
```

Comment:
This is used only by the 2 ActiveDocument services and now that they're in the same assembly, there is no need for this to be public anymore.